src/main/kotlin/fi/metatavu/metaform/server/rest/AdminThemeApi.kt

```diff
@@ -0,0 +1,74 @@
+package fi.metatavu.metaform.server.rest
+
+import fi.metatavu.metaform.api.spec.model.AdminTheme
+import fi.metatavu.metaform.server.controllers.AdminThemeController
+import java.util.UUID
+import javax.enterprise.context.RequestScoped
+import javax.inject.Inject
+import javax.transaction.Transactional
+import javax.ws.rs.core.Response
+
+@RequestScoped
+@Transactional
+class AdminThemeApi : fi.metatavu.metaform.api.spec.AdminThemeApi, AbstractApi() {
+    @Inject
+    lateinit var adminThemeController: AdminThemeController
+
+    override suspend fun createAdminTheme(adminTheme: AdminTheme): Response {
+        val userId = loggedUserId ?: return createForbidden(UNAUTHORIZED)
+        if (!isRealmMetaformAdmin) {
+            return createForbidden(createNotAllowedMessage(CREATE, ADMIN_THEME))
+        }
+
+        return createOk(adminThemeController.create(
+            UUID.randomUUID(),
+            adminTheme.data,
+            adminTheme.name,
+            adminTheme.slug,
+            userId,
+            userId,
```

Comment:
```suggestion
            userId
```
I think it's a duplicate?